index.html

```diff
@@ -10,59 +10,79 @@
+      </div>
+      <div class="language">
+        <div class="world">
+          <img src="./src/assets/world.png" alt="world icon">
         </div>
-        <datalist id="repositories_history">
-          <!-- TODO: ADD repositories history on this datalist -->
-        </datalist>
-        <datalist id="users_history">
-          <!-- TODO: ADD users history on this datalist -->
-        </datalist>
+        <div class="content-select">
+          <select>
+            <option default>Select language</option>
+            <option>English</option>
+            <option>Spanish</option>
+          </select>
+        </div>        
+      </div>
+    </div>
+    <div class="form">
+      <div style="display: block">
         <div class="div">
-          <button id="redirectToSearch" class="btn">Ir</button>
-          <button id="copySearch" class="btn btn--secondary">Copiar</button>
+          <img src="./src/assets/git.png" width="100px" height="100px"/>
         </div>
-    </form>
-  </div>
-</div>
+        <form id="calendary_filter" style="margin-top: 15px" class="calendary_filter">
+            <select id="input_condition" name="condition" class="input">
+              <option default>
+                Select
+              </option>
+              <option value="after">
+                After of
+              </option>
+              <option value="before">
+                Before of
+              </option>
+              <option value="equals">
+                Equals to
+              </option>
+            </select>
+            <!-- <input 
+              type="text" 
+              placeholder="User/Organization" 
+              list="users_history"
+              id="input_user" 
+              name="user" 
+              class="input calendary_filter__repository_name" 
+              autofocus
+            />
+            <input 
+              type="text" 
+              placeholder="Repository name" 
+              list="repositories_history"
+              id="input_repository" 
+              name="repository" 
+              class="input calendary_filter__repository_name" 
+            /> -->
+            <div>
+              <button id="goback" class="btn--back">
+                <img src="./src/assets/arrow.png" style="margin-top: 3px" width="15px" height="15px" alt="Back button icon"/>
+              </button>
+              <input type="date" name="date" id="input_date" class="calendary_filter__date_search" />
+            </div>
+            <datalist id="repositories_history">
+              <!-- TODO: ADD repositories history on this datalist -->
+            </datalist>
+            <datalist id="users_history">
+              <!-- TODO: ADD users history on this datalist -->
+            </datalist>
+            <div class="div">
+              <button id="redirectToSearch" class="btn">Ir</button>
+              <button id="copySearch" class="btn btn--secondary">Copiar</button>
+            </div>
+        </form>
```

Comment:
Check indentation please

Reply:
I don't understand this, can you suggest me a code?